docs/source/markdown/podman-pull.1.md

```diff
@@ -79,12 +79,13 @@ This is a Docker specific option to disable image verification to a Docker
 registry and is not supported by Podman.  This flag is a NOOP and provided
 solely for scripting compatibility.
 
+**--override-arch**=*ARCH*
+Override the machine's default architecture of the image to be pulled. For example, `arm`.
+
 **--override-os**=*OS*
 Use OS instead of the running OS for choosing images
 
-**--override-arch**=*ARCH*
-
-Override the machine's default architecture of the image to be pulled. For example, `arm`.
+**--override-variant** _VARIANT_ Use _VARIANT_ instead of the running architecture variant for choosing images.
```

Comment:
This looks like if should be on two lines instead of one, also should have a `=` between the flag and the value

Reply:
Also, maybe a bit more details on what a variant is, given that we already have an override-arch flag to select architecture.